templates/zerver/help/import-your-settings.md

```diff
@@ -0,0 +1,38 @@
+# Import your settings
+
+When you create a Zulip account using an email address already associated with
+an account in another Zulip organization, you can import your user settings
```

Comment:
Should we clarify that this other organization is on the same Zulip server? We could have this say "another Zulip organization on Zulip Cloud or the same self-hosted Zulip installation" for clarity; not sure how to tighten that without being confusing.

Reply:
Yeah I also think this is probably the best way to phrase it:

<img width="771" alt="image" src="https://user-images.githubusercontent.com/2343554/189466425-6dae9e81-8a15-4dbd-9b09-9f42ee8b6a2b.png">

Reply:
That WFM. @alya FYI.